src/lib/plugins/sast/checks.ts

```diff
@@ -23,3 +28,18 @@ export async function getSastSettingsForOrg(org): Promise<SastSettings> {
 
   return (response as any).body;
 }
+
+export async function trackUsage(org): Promise<TrackUsageResponse> {
+  const response = await request({
+    method: 'POST',
+    headers: {
+      Authorization: `token ${snyk.api}`,
```

Comment:
I think using a getter is safer here. It's defined in `api-token.ts` 

```suggestion
      Authorization: `token ${getApiToken()}`,
```